package.json

```diff
@@ -214,9 +214,11 @@
     "ssh2": "^0.6.1",
     "vscode-express": "^1.0.1",
     "vscode-extension-telemetry": "^0.1.0",
+    "vscode-iot-device-cube-sdk": "^0.0.18",
     "winreg": "^1.2.3"
   },
   "extensionDependencies": [
-    "vsciot-vscode.azure-iot-toolkit"
+    "vsciot-vscode.azure-iot-toolkit",
+    "zlhe-test.vscode-iot-device-cube"
```

Comment:
TODO: Move the `vscode-iot-device-cube` extension to vscode official account.@dilin